var/spack/repos/builtin/packages/kokkos/package.py

```diff
@@ -416,8 +416,12 @@ def test_run(self):
             raise SkipTest(f"{cmake_path} is missing")
 
         cmake = self.spec["cmake"].command
-        cmake(cmake_path, "-DEXECUTABLE_OUTPUT_PATH=" + cmake_path)
+        cmake_args = ["-DEXECUTABLE_OUTPUT_PATH=" + cmake_path]
+        if self.spec.satisfies("+rocm"):
+            prefix_paths = ";".join(spack.build_environment.get_cmake_prefix_path(self))
+            cmake_args.append("-DCMAKE_PREFIX_PATH={0}".format(prefix_paths))
```

Comment:
FYI. We are encouraging maintainers to convert from string `format()` to f-strings (https://docs.python.org/3/tutorial/inputoutput.html) as we do the same in the core.